src/components/Reportees/index.jsx

```diff
@@ -1,56 +1,78 @@
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+import CONFIG from '../../config';
 
-const styles = ({
+const styles = theme => ({
   root: {
-    margin: '0 0.5rem 0 0',
-  },
-  header: {
-    margin: '0.5rem 0 0.5rem 0',
-  },
-  icon: {
-    fontSize: '1rem',
-    verticalAlign: 'bottom',
-  },
-  person: {
-    // This makes it line up better with the table of components
-    padding: '1px',
-    display: 'flex',
+    width: '50%',
+    marginTop: theme.spacing.unit * 3,
+    overflowX: 'auto',
   },
 });
 
 const sortByPersonName = (a, b) => (a.cn <= b.cn ? -1 : 1);
 
 const Reportees = ({
-  classes, ldapEmail, partialOrg, onPersonDetails,
+  classes, ldapEmail, partialOrg, metrics,
 }) => (
-  <div className={classes.root}>
-    <div height="1rem">&nbsp;</div>
-    {Object.values(partialOrg)
-      .filter(({ cn }) => cn !== ldapEmail)
-      .sort(sortByPersonName)
-      .map(({ cn, mail }) => (
-        <div key={mail} className={classes.person}>
-          <DrilldownIcon
-            name={mail}
-            onChange={onPersonDetails}
-            properties={{
-              ldapEmail: mail,
-            }}
-          />
-          <span>{`${cn} `}</span>
-        </div>
-      ))}
-  </div>
+  <Paper className={classes.root}>
+    <Table className={classes.table}>
+      <TableHead>
+        <TableRow>
+          <TableCell />
+          {Object.values(CONFIG.reporteesMetrics).map(({ label }) => (
+            <TableCell key={label}>{label}</TableCell>
+          ))}
+        </TableRow>
+      </TableHead>
+      <TableBody>
+        {Object.values(partialOrg)
+          .filter(({ cn }) => cn !== ldapEmail)
+          .sort(sortByPersonName)
+          .map(({ cn, mail, bugzillaEmail }) => (
+            <TableRow key={mail}>
+              <TableCell className={classes.leftColumn} key={mail}>{`${cn} `}</TableCell>
+              {Object.keys(CONFIG.reporteesMetrics).map((metricUid) => {
+                const countLink = ((metrics || {})[bugzillaEmail] || {})[metricUid];
+                return (
+                  <TableCell align="right" key={metricUid}>
+                    {countLink && (
+                      <a
+                        key={countLink.link}
+                        href={countLink.link}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                        className={classes.link}
+                      >
+                        {countLink.count}
+                      </a>
+                    )}
+                  </TableCell>
+                );
+              })}
+            </TableRow>
+          ))}
+      </TableBody>
+    </Table>
+  </Paper>
 );
 
 Reportees.propTypes = {
   classes: PropTypes.shape({}).isRequired,
   ldapEmail: PropTypes.string.isRequired,
   partialOrg: PropTypes.shape({}).isRequired,
-  onPersonDetails: PropTypes.func.isRequired,
+  metrics: PropTypes.shape({}),
```

Comment:
`PropTypes.shape(PropTypes.object),`

Reply:
If I do that change I get this very odd behaviour when running tests (its suddenly considered as 'required'):

```
Warning: Failed prop type: The prop `metrics.isRequired` is marked as required in `Reportees`, but its value is `undefined`.
```